packages/react-json-tree/src/index.tsx

```diff
@@ -7,12 +7,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import JSONNode from './JSONNode';
 import createStylingFromTheme from './createStylingFromTheme';
-import {
+import type {
   invertTheme,
-  type StylingConfig,
-  type StylingFunction,
-  type StylingValue,
-  type Theme,
+  StylingConfig,
+  StylingFunction,
+  StylingValue,
+  Theme,
 } from 'react-base16-styling';
```

Comment:
You'll need to split this into two separate import statements, one for the types and one for `invertTheme` which is not a type.

Reply:
Hey @Methuselah96 I have made the requested changes. Review it now.
Thanks!